common/changes/office-ui-fabric-react/commandBar-styles-cleanup_2018-05-14-23-07.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "CommandBar: Removed unnecessary endAligned styleprops and styles",
+      "type": "patch"
```

Comment:
minor really...it was in experiments before. the prop has never been released to the public.

Reply:
The "public" has been consuming the experiments version of the component though... Hm.... We should do a final scrub of all Command Bar change files to make sure it makes sense when they land .. (e.g. no confusion over which "Command Bar" and make sure they all look sensible side by side - in that regard since Command Bar promotion from experiments to the main OUFR package is already a major change, we want this change to sit alongside too).

Reply:
I'm not sure the change files are at all needed; I think we should really just track major changes in the release notes and then manually put that into a change file right before publish.

Reply:
Sounds like we're saying the same things :) (maybe two slightly different approaches to accomplishing the same thing).